trio/__init__.py

```diff
@@ -14,49 +14,85 @@
 
 __all__ = []
 
-from ._toplevel_core_reexports import *
+from ._toplevel_core_reexports import (TrioInternalError,
+                                        RunFinishedError,
+                                        WouldBlock,
+                                        Cancelled,
+                                        ResourceBusyError,
+                                        ClosedResourceError,
+                                        MultiError,
+                                        run,
+                                        open_nursery,
+                                        open_cancel_scope,
+                                        current_effective_deadline,
+                                        TASK_STATUS_IGNORED,
+                                        current_time,
+                                        TaskLocal,
+                                        __all__)
+
```

Comment:
Not sure what the trio style is but rather than have towers of arguments at various arbitrary columns I personally prefer to drop down to the next line and only indent one level:
```python
from ._toplevel_core_reexports import (
    ClosedResourceError,
    MultiError,
    ResourceBusyError,
    RunFinishedError,
    TrioInternalError,
    Cancelled,
    WouldBlock,
    TaskLocal,
    TASK_STATUS_IGNORED,
    current_effective_deadline,
    current_time,
    open_cancel_scope,
    open_nursery,
    run,
)
```
...this also allows for a more compact form:
```python
from ._toplevel_core_reexports import (
    ClosedResourceError, MultiError, ResourceBusyError, RunFinishedError,
    TrioInternalError, Cancelled, WouldBlock, TaskLocal, TASK_STATUS_IGNORED,
    current_effective_deadline, current_time, open_cancel_scope, open_nursery,
    run,
)
```

Reply:
Yep, the compact form is [enforced by `yapf`](https://travis-ci.org/python-trio/trio/jobs/413179559#L2944); see my comment above re: Travis.

Reply:
Yapf allows either of the two forms @dhirschfeld showed – IIRC you get the first one if you put a trailing comma on the last item, and the "more compact" one if you don't use a trailing comma.